.github/workflows/release.yml

```diff
@@ -96,6 +96,8 @@ jobs:
         if: runner.os == 'macOS'
         run: |
           sudo xcode-select -s "/Applications/Xcode_15.1.app"
+          # Install lld
+          brew install lld
```

Comment:
just fyi, this installs llvm 19. i'm not sure if it will cause any compatibility problems. leaving it as a small note.

Reply:
You're right. This should be fine as the release build job doesn't use homebrew LLVM at all, the bazel toolchain is configured solely based on the clang/llvm available in the Xcode installation. Installing homebrew LLVM 19 instead of the default version should not be an issue (I initially included this command in the test job too to try it out, seems to have worked flawlessly. I primarily want to install LLD to make release builds smaller, so this is not needed for the regular test run).

Reply:
Do we need to add this install to the README?

Reply:
This will only be needed for macOS release builds. Developers generally shouldn't need to build those regularly, but I think it should still be supported without needing more stuff – ended up removing LLD from release_macos and manually adding it in the release job instead.